certification/internal/shell/scorecard_basic-check-spec.go

```diff
@@ -0,0 +1,123 @@
+package shell
+
+import (
+    "os/exec"
+    "os"
+    "io/ioutil"
+    "path/filepath"
+
+    "github.com/komish/preflight/certification"
+    "github.com/sirupsen/logrus"
+)
+
+type ScorecardBasicSpecCheck struct{}
+
+func (p *ScorecardBasicSpecCheck) Validate(bundle string, logger *logrus.Logger) (bool, error) {
+    
+    //Curentdir
+    homedir, err := os.Getwd()
+    if err != nil {
+        logger.Error("unable to get current directory: ", err)
+        return false, err
+    }
+
+    artifactsdir := filepath.Join(homedir, "/artifacts")
+    err = os.MkdirAll(artifactsdir, 0777)
+    if err != nil {
+        logger.Error("unable to create artifactsdir: ", err)
+        return false, err
+    }
+
+    scorecardresult := "operator_bundle_scorecard_BasicSpecCheck.json"
```

Comment:
This declaration could be in a `const` block.

Reply:
Done.